files/usr/share/cinnamon/applets/notifications@cinnamon.org/applet.js

```diff
@@ -314,13 +315,12 @@ function timeify(orig_time) {
             str += _(" (Just now)");
             break;
         case (diff > 15 && diff <= 59):
-            str += _(" (%s seconds ago)").format(diff.toString());
+            diff_seconds = diff.toString();
+            str += Gettext.ngettext(" (%s second ago)", " (%s seconds ago)", diff_seconds).format(diff_seconds);
```

Comment:
I think we can save a line here, because ngettext takes a number:

    str += Gettext.ngettext(" (%s second ago)", " (%s seconds ago)", diff).format(diff.toString());

And maybe we can avoid calling toString() manually be using the  % notation below, in combination with %d?